src/App.tsx

```diff
@@ -15,71 +17,94 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  HandlerSelectedGoods = () => {
```

Comment:
```suggestion
  handleSelectedGood = () => {
```
Use camelCase. Also, name your function correctly
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364